RecoTracker/TkSeedingLayers/test/SeedingHitSet_t.cpp

```diff
@@ -0,0 +1,80 @@
+#include "RecoTracker/TkSeedingLayers/interface/SeedingHitSet.h"
```

Comment:
Great to see a unit test! Please consider using catch2 system as it provides a richer set of testing abilities: https://github.com/catchorg/Catch2

We make use of Catch2 throughout the framework for our unit tests.

Reply:
Ah! I see, thanks. Sure, let me have a look at it to restructure the test.

Reply:
I've mimicked some other tests around the framework. Should be fine.